app/javascript/components/App/index.js

```diff
@@ -17,26 +17,26 @@ const MuiTheme = {
   },
 }
 
-const App = ({ loading, currentUser, offices, userPopover, toggleUserPopover, updateUserOffice, children }) => (
-  <UserContext.Provider value={currentUser}>
-    <MuiThemeProvider muiTheme={getMuiTheme(MuiTheme)}>
-      {loading ? (
-        <Loading />
-      ) : (
-        <div>
-          <Header
-            currentUser={currentUser}
-            offices={offices}
-            togglePopover={e => toggleUserPopover(e.currentTarget)}
-            popover={userPopover}
-            handleOfficeSelect={office => updateUserOffice(currentUser, office)}
-          />
-          <WelcomeModal />
-          {children}
-        </div>
-      )}
-    </MuiThemeProvider>
-  </UserContext.Provider>
-)
+const App = ({ loading, currentUser, offices, userPopover, toggleUserPopover, children }) => {
+  if (!currentUser) return null
+
+  return (
+    <UserContextProvider user={currentUser}>
```

Comment:
`currentUser &&` instead of `if (!currentUser) return null` ?

Reply:
I usually only use the `&&` technique when controlling rendering inside JSX.